doc/man/bitcoin-qt.1

```diff
@@ -514,7 +514,7 @@ Password for JSON\-RPC connections
 .IP
 Username and hashed password for JSON\-RPC connections. The field
 <userpw> comes in the format: <USERNAME>:<SALT>$<HASH>. A
-canonical python script is included in share/rpcuser. The client
+canonical python script is included in share/rpcauth. The client
```

Comment:
.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.

Reply:
Fixing